feat(CRED-83): WIP: Show charge amount, client id and charge amount o…

kayr opened the pull request:
Read for review

kayr committed:
feat(CRED-83): WIP: Show charge amount, client id and charge amount on checker make inbox

kayr committed:
feat(CRED-83): Fix failing test

happyFiter reviewed (approved):
LGTM

happyFiter commented on custom/crediblex/infrastructure/commands/src/main/java/com/crediblex/fineract/commands/CredXAuditReadPlatformService.java:
```diff
@@ -0,0 +1,110 @@
+        // Create a lookup map for quick access
+        @SuppressWarnings("Convert2MethodRef")
+        Map<Long, LoanChargeWaiveDetails.Result> detailsMap = waiveDetails.stream()
+                .collect(toMap(c -> c.getLaonChargeId(), c -> c));
```
take a look at the typo: getLaonChargeId() likely should be getLoanChargeId()

kayr replied:
you have a sharp eye... noted.. lemme fix

kayr replied:
Fixed

happyFiter commented on custom/crediblex/infrastructure/commands/src/main/java/com/crediblex/fineract/commands/repository/EzySqlLoanChargeWaiverRepository.java:
```diff
@@ -0,0 +1,32 @@
+        LoanChargeWaiveDetails Q = loanChargeWaiveDetails();
+        return ezySql
+                    .from(Q)
+                    .where(Q.LAON_CHARGE_ID.in(chargeIds))
```
take a look at the typo in LAON_CHARGE_ID

happyFiter replied:
I believe EzySql will handle exceptions gracefully, i'm not sure it's guarded against Type mismatch upon returns?

kayr replied:
It will fail just like how JDBC would... i.e if you defines a column in you POJO as an integer.. but it is actually a string then the type assignment would fail. Not sure if that answers your question?

The ideas is that your SQL queries should always consistently return the same datatype and in that case the result assignment would never fail.  If it not consistent then it might might fail. 

e.g 
```sql
select name, age from person
```

should always return 
```java
class Person{
   String name
   Integer age
}
```

happyFiter replied:
Looks good

kayr committed:
Merge branch 'develop' into feature/CRED-83

kayr committed:
feat(CRED-83): Correct loanChargeId typo and bump ezyquery version

- Fixed typos (`laonChargeId` to `loanChargeId`) across the codebase.
- Upgraded `ezyquery` plugin to version `0.0.21`.
